filecoin-project/sentinel-visor
# feat: create a long running daemon mode for visor (lily)

…ply and revert events (#408)

chore: reuse watcher in lily watch command (#422)

Merge master into lily (#425)

* fix: ignore false positive gosec failure in wait package (#421)

* fix: embed genesis block in executable (#420)

* feat: add benchmarking of vectors (#423)

* feat: add benchmarking of vectors

* feat: add benchmarking of vectors

* Use nicer test and benchmark names

fix(linter): make deps before linting in CI (#437)

merge master into lily (#432)

* fix(indexer): only close TIpSetIndexer in walk and watch
* chore: increase linter timeout in ci

scheduler job control + watch, walk, and job cli (#414)

merge master into lily (#442)

* fix: ignore false positive gosec failure in wait package (#421)

* fix: embed genesis block in executable (#420)

* feat: add benchmarking of vectors (#423)

* feat: add benchmarking of vectors

* feat: add benchmarking of vectors

* Use nicer test and benchmark names

* Chore/sqlotus dag prefetch option (#430)

* Disable preloader by default, toggle with LOTUS_CHAINSTORE_PRELOAD_RECENTS

* Switch to a cli string as suggested by @iand

* Remove one more inefficiency for the R/O lens

https://github.com/filecoin-project/go-bs-postgres-chainnotated/commit/b8b5d500e368d2ed

* Forgotten option/helptext spec

* Add support for static Get() prefetch

* close TipSetObs in walker and watcher

* fix(indexer): only close TIpSetIndexer in walk and watch

- subsequent calls to close will hang forever

* chore: increase linter timout in ci

* chore: update for Lotus 1.5.3 (#439)

* chore: update for Lotus 1.5.3

* fixup! chore: update for Lotus 1.5.3

* Update statediff version

Co-authored-by: frrist <forrest@protocol.ai>

Co-authored-by: Ian Davis <nospam@iandavis.com>
Co-authored-by: Peter Rabbitson <ribasushi@protocol.ai>

chore: remove confidence argument from lily walk command (#444)

chore: reorganise visor commands (#445)

feat: add storage configuration to daemon (#446)

* feat: add storage configuration to daemon

* Start wiring up storage to config

* Finish wiring up storage config

* check error return when ensuring config

* Close config file on error

feat: Add --api parameter to daemon client commands (#450)

* feat: Add --api parameter to daemon client commands

* Make api default value a valid multiaddr

* Add api token parameter

Merge master into lily (#463)

* fix: ignore false positive gosec failure in wait package (#421)

* fix: embed genesis block in executable (#420)

* feat: add benchmarking of vectors (#423)

* feat: add benchmarking of vectors

* feat: add benchmarking of vectors

* Use nicer test and benchmark names

* Chore/sqlotus dag prefetch option (#430)

* Disable preloader by default, toggle with LOTUS_CHAINSTORE_PRELOAD_RECENTS

* Switch to a cli string as suggested by @iand

* Remove one more inefficiency for the R/O lens

https://github.com/filecoin-project/go-bs-postgres-chainnotated/commit/b8b5d500e368d2ed

* Forgotten option/helptext spec

* Add support for static Get() prefetch

* close TipSetObs in walker and watcher

* fix(indexer): only close TIpSetIndexer in walk and watch

- subsequent calls to close will hang forever

* chore: increase linter timout in ci

* chore: update for Lotus 1.5.3 (#439)

* chore: update for Lotus 1.5.3

* fixup! chore: update for Lotus 1.5.3

* Update statediff version

Co-authored-by: frrist <forrest@protocol.ai>

* Release notes for v0.5.5 (#440)

* chore: update lotus to 1.6.0 (#443)

* chore: upgrade lotus dependency to 1.6.0-rc1 (#447)

* chore: update lotus to 1.6.0 (#455)

* Release notes for v0.5.6 (#456)

* fix: ensure persistence semaphore channel is drained on close (#457)

* fix: ensure persistence semaphore channel is drained on close

* Add hidden window flag for testing

* Reduce actor happy path logging

* Make sure actor errors reported are appended to correct list

* Release notes for v0.5.7 (#458)

* Update to latest lotus commit

* Add missing support/tools/go.sum

* Fix merge bugs in makefile

* Fix merge bugs in makefile

Co-authored-by: Peter Rabbitson <ribasushi@protocol.ai>
Co-authored-by: frrist <forrest@protocol.ai>

feat: clean up command line flags for single-run jobs (#464)

* feat: clean up command line flags for single-run jobs
* Fix dockerfile

Co-authored-by: frrist <forrest@protocol.ai>

polish(lily): add job name flag (#467)

- closes #452

polish(job): print job type, params, and errors in list (#468)

- closes #451

fix: ensure event channel is initialised before watch starts (#471)

chore: merge master into lily branch (#472)

* fix: ignore false positive gosec failure in wait package (#421)

* fix: embed genesis block in executable (#420)

* feat: add benchmarking of vectors (#423)

* feat: add benchmarking of vectors

* feat: add benchmarking of vectors

* Use nicer test and benchmark names

* Chore/sqlotus dag prefetch option (#430)

* Disable preloader by default, toggle with LOTUS_CHAINSTORE_PRELOAD_RECENTS

* Switch to a cli string as suggested by @iand

* Remove one more inefficiency for the R/O lens

https://github.com/filecoin-project/go-bs-postgres-chainnotated/commit/b8b5d500e368d2ed

* Forgotten option/helptext spec

* Add support for static Get() prefetch

* close TipSetObs in walker and watcher

* fix(indexer): only close TIpSetIndexer in walk and watch

- subsequent calls to close will hang forever

* chore: increase linter timout in ci

* chore: update for Lotus 1.5.3 (#439)

* chore: update for Lotus 1.5.3

* fixup! chore: update for Lotus 1.5.3

* Update statediff version

Co-authored-by: frrist <forrest@protocol.ai>

* Release notes for v0.5.5 (#440)

* chore: update lotus to 1.6.0 (#443)

* chore: upgrade lotus dependency to 1.6.0-rc1 (#447)

* chore: update lotus to 1.6.0 (#455)

* Release notes for v0.5.6 (#456)

* fix: ensure persistence semaphore channel is drained on close (#457)

* fix: ensure persistence semaphore channel is drained on close

* Add hidden window flag for testing

* Reduce actor happy path logging

* Make sure actor errors reported are appended to correct list

* Release notes for v0.5.7 (#458)

* fix(tasks): use correct ActorInfo for actor state extraction (#449)

* fix(tasks): use corret actor info for actor state extraction
- fixes #273

* fix(lens): remove OptimizedStateGetActorWithFallback
- fixes #405

* fix(market-task): fix market actor task and gen vector
- fixes #465

* polish(vector): support msapproval model validation

* test(vector): generate new test vectors

* ci: increase instance size due to test vectors

* Add MinerSectorInfo for extended sectors (#448)

Fixes #441. Extended sectors have new deadlines but this is not recorded anywhere.

* Fix makefile conflicts

Co-authored-by: Peter Rabbitson <ribasushi@protocol.ai>
Co-authored-by: frrist <forrest@protocol.ai>
Co-authored-by: Hector Sanjuan <hector@protocol.ai>
